docker/docker-build.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+docker build -t rock64/rock64-arch-linux-build:latest .
```

Comment:
I'd maybe rename this script to `build-docker-image.sh` to clarify the intent

Reply:
Maybe can do also the same to make the script deterministic no matter where the user is running it from.

Reply:
Sounds good!